.github/workflows/wallet-ext-prs.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     if: needs.diff.outputs.isWalletExt == 'true'
     runs-on: ubuntu-latest
     env:
-      working-directory: ./wallet
+      working-directory: ./apps/wallet
```

Comment:
@Jordan-Mysten it's so cool that this is all OSS. I'm just scrolling the GitHub history of wallet on my phone 😂